pkg/apis/pipeline/v1alpha1/register.go

```diff
@@ -48,6 +48,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
 		&Task{},
 		&TaskList{},
+		&Condition{},
+		&ConditionList{},
```

Comment:
whoa i never would have known to do this!

Reply:
Found out the hard way while writing an e2e test :sweat_smile: